pallets/admin-utils/tests/mock.rs

```diff
@@ -429,6 +429,14 @@ impl pallet_admin_utils::SubtensorInterface<AccountId, Balance, RuntimeOrigin> f
     fn set_weights_min_stake(min_stake: u64) {
         SubtensorModule::set_weights_min_stake(min_stake);
     }
+
+    fn set_nominator_min_required_stake(min_stake: u64) {
+        SubtensorModule::set_nominator_min_required_stake(min_stake);
+    }
+
+    fn clear_small_nominations() {
+        SubtensorModule::clear_small_nominations();
+    }
```

Comment:
DQ but why do you use this pattern of wrapping the `Subtensor` module calls instead of calling them directly in the code?

Reply:
We could do it that way , but chose to follow the pattern in the repo. Maybe we can address this in a separate Pr if we want to change it ?

Reply:
Yeah we can ask @unconst maybe why that decision was made. Def not something to do in this pr.